Modules/Core/Controllers/MailerController.php

```
@@ -2,160 +2,187 @@
-     *                               - quote: the quote model instance
+     * @legacy-function quote
+     * @legacy-file application/modules/mailer/controllers/Mailer.php
      */
     public function quote(Request $request, int $quote_id)
     {
-        if ( ! $this->mailer_configured) {
+        if (!$this->mailer_configured) {
             return;
         }
+
         $email_template_id = get_setting('email_quote_template');
```

PHPMD (2.15.0)

The variable $email_template_id is not named in camelCase. (undefined)

(CamelCaseVariableName)